tests/schema/tests.py

```diff
@@ -2403,6 +2403,20 @@ class Meta:
         student = Student.objects.create(name='Some man')
         doc.students.add(student)
 
+    @unittest.skipUnless(connection.vendor == 'oracle', 'Oracle specific sequence name truncation')
+    def test_oracle_sequence_name_truncation_regression(self):
+        """
+        Regression test for #28451.
```

Comment:
Docstrings should state the expected behavior. Ticket references should be reserved for obscure issues (not needed here). These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).